metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/extractor/TaxonomyExtractor.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.importer.extractor;
+
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Taxonomy;
+import uk.ac.ebi.ampt2d.metadata.persistence.repositories.TaxonomyRepository;
+
+public class TaxonomyExtractor {
+    private static final Long TAXONOMY_ID = new Long(1);
+
+    private static Taxonomy taxonomy;
+
+    public TaxonomyExtractor(TaxonomyRepository taxonomyRepository) {
+        taxonomy = taxonomyRepository.save(new Taxonomy(9606,"Homo Sapiens"));
+    }
+
+    public Taxonomy getTaxonomy() {
```

Comment:
Could the interface for this class be made closer to what the final one should look like? That will give a better idea of how other components will use it, right now it is too trivial because it is not parameterized.